src/tensorflow/lite/micro/tools/make/.gitignore

```diff
@@ -0,0 +1 @@
+downloads
```

Comment:
the micro/tools/make folder should not exist in the Arduino repo.

Reply:
removed.